docs/changelog/1493.feature.rst

```diff
@@ -0,0 +1 @@
+Add ``interrupt_timeout`` and ``terminate_timeout`` that configure delay between SIGINT, SIGTERM and SIGKILL when tox is interrupted. - by :user:`sileht`
```

Comment:
you can use the `` :conf:`interrupt_timeout ` `` to make them references please do so

Reply:
done